hazelcast/src/main/java/com/hazelcast/persistence/PersistenceService.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.persistence;
+
+/**
+ * Service for interacting with Hot Restart. For example - starting cluster-wide hot restart data backups, determining the
```

Comment:
Need to eliminate "hot restart"/"hot backup" references from the javadoc as well eg

```suggestion
 * Service for interacting with Persistence. For example - starting cluster-wide data backups, determining the
```